### PR TITLE
Hardcode h264 parameters

### LINKED ISSUE #1570: Hardcode h264 parameters
We have three Ansible vars related to uStreamer's H264 settings that can be deleted and replaced with hardcoded values:

* `ustreamer_h264_sink`
* `ustreamer_h264_sink_mode`
* `ustreamer_h264_sink_rm`

Deleting these vars impacts a few files:

* We're [conditioning a Janus install](https://github.com/tiny-pilot/tinypilot/blob/b9d60675e8bbf0d112614d8d48e328267cf80fd7/ansible-role-ustreamer/tasks/main.yml#L15) on these vars, but we should unconditionally install Janus.
* We're [conditioning uStreamer flags](https://github.com/tiny-pilot/tinypilot/blob/b9d60675e8bbf0d112614d8d48e328267cf80fd7/ansible-role-ustreamer/files/launch#L105-L107) but we should unconditionally add those flags [in this section](https://github.com/tiny-pilot/tinypilot/blob/49f0fc9d95302048feb575b37bd2c0a1723e3752/ansible-role-ustreamer/files/launch#L88-L90)
* We're [templating `janus.plugin.ustreamer.jcfg`](https://github.com/tiny-pilot/tinypilot/blob/b9d60675e8bbf0d112614d8d48e328267cf80fd7/ansible-role-ustreamer/templates/janus.plugin.ustreamer.jcfg.j2) with `ustreamer_h264_sink` but we can replace it with the hardcoded string of `tinypilot::ustreamer::h264`.

## Background

See the [Ansible-to-Debian refactoring doc](https://github.com/tiny-pilot/tinypilot-pro/blob/master/ANSIBLE.md). You should already have done an Ansible-to-Debian refactoring PR before taking on this one.

### PR DESCRIPTION
**Pull request by db39**
Resolves #1570

This change removes uStreamer variables in the Ansible role and hardcodes them. We now also unconditionally install Janus.

To test this bundle, run:

```bash
curl \
  --silent \
  --show-error \
  --location \
  https://raw.githubusercontent.com/tiny-pilot/tinypilot/master/scripts/install-bundle | \
  sudo bash -s -- \
    https://output.circle-artifacts.com/output/job/8025923f-9ba2-4e45-aef0-952f0f12655b/artifacts/0/bundler/dist/tinypilot-community-20230824T1222Z-1.9.0-62+3908e48.tgz
```

<a data-ca-tag href="https://codeapprove.com/pr/tiny-pilot/tinypilot/1585">Review on CodeApprove</a>

### CODE REVIEW

**Comment by db39** [comment]
I've removed the uStreamer variables and hardcoded the values where required. It works as expected.

I've left the '[determine whether to install Janus](https://github.com/tiny-pilot/tinypilot/blob/30f8f6e35b2d0907babd22b02af20a8ede2f60ca/ansible-role-ustreamer/tasks/main.yml#L40)' task in place since `ustreamer_install_janus` gets used [elsewhere](https://github.com/tiny-pilot/tinypilot/blob/30f8f6e35b2d0907babd22b02af20a8ede2f60ca/ansible-role-ustreamer/tasks/main.yml#L96). We could remove all references to `ustreamer_install_janus`, but we'd have to consider the '[decide whether to support audio streaming](https://github.com/tiny-pilot/tinypilot/blob/30f8f6e35b2d0907babd22b02af20a8ede2f60ca/ansible-role-ustreamer/tasks/main.yml#L47)' task which currently relies on the uStreamer install. We could hardcode this too.

**Review by mtlynch — CHANGES_REQUESTED** [review]
##### Automated comment from [CodeApprove ➜](https://codeapprove.com/pr/tiny-pilot/tinypilot/1585)



<hr />

In: **Discussion**
>I've left the 'determine whether to install Janus' task in place since ustreamer_install_janus gets used elsewhere. We could remove all references to ustreamer_install_janus, but we'd have to consider the 'decide whether to support audio streaming' task which currently relies on the uStreamer install. We could hardcode this too.

Yeah, I'd like to tear all of it out. `ustreamer_install_janus` always evaluates to `True` currently, but in a convoluted way. We want to strip out the variable and simplify the code to apply the same logic as when `ustreamer_install_janus=True`.

In this PR, we want to remove all references to all of these Ansible vars:

* `ustreamer_install_janus`
* `ustreamer_h264_sink`
* `ustreamer_h264_sink_mode`
* `ustreamer_h264_sink_rm`

In other words, `grep ustreamer_h264_sink ./ -R` should return 0 matches (ditto for the rest).

<hr />

In: **ansible-role-ustreamer/files/launch**:

```
> Line 90
USTREAMER_ARGS+=('--persistent')
```
We don't want to delete this flag.

<hr />

In: **ansible-role-ustreamer/files/launch**:

```
> Line 92
USTREAMER_ARGS+=('--h264-sink-rm' 'yes')
```
It should just be `USTREAMER_ARGS+=('--h264-sink-rm')`


<hr />

👀 @db39 it's your turn please take a look

**Review by mtlynch — CHANGES_REQUESTED** [review]
##### Automated comment from [CodeApprove ➜](https://codeapprove.com/pr/tiny-pilot/tinypilot/1585)



<hr />

In: **Discussion**
A good sanity check on this PR is to verify that uStreamer's command-line flags are the same after applying the changes. You can see them in the logs:

```text
Aug 24 01:21:52 tinypilot launch[559]: + exec /opt/ustreamer/ustreamer --port 48001 --encoder m2m-image --format uyvy --workers 3 --drop-same-frames 30 --persistent --dv-timings --h264-sink tinypilot::ustreamer::h264 --h264-sink-mode 777 --h264-sink-rm
```

The order of the flags doesn't have to stay exactly the same, but we should have the same set of flags before and after this refactoring.


<hr />

👀 @db39 it's your turn please take a look

**Review by db39 — COMMENTED** [review]
##### Automated comment from [CodeApprove ➜](https://codeapprove.com/pr/tiny-pilot/tinypilot/1585)



<hr />

In: **Discussion**
Makes sense! Here's that line of the log from my most recent test:

```text
Aug 24 13:08:28 raspberrypi launch[1247]: + exec /opt/ustreamer/ustreamer --port 48001 --persistent --h264-sink tinypilot::ustreamer::h264 --h264-sink-mode 777 --h264-sink-rm --encoder m2m-image --format uyvy --workers 3 --drop-same-frames 30 --dv-timings
```

<hr />

In: **Discussion**
Resolved

<hr />

In: **ansible-role-ustreamer/files/launch**:
Resolved

<hr />

In: **ansible-role-ustreamer/files/launch**:

```
> Line 90
USTREAMER_ARGS+=('--persistent')
```
Resolved


<hr />

👀 @mtlynch it's your turn please take a look

**Review by mtlynch — CHANGES_REQUESTED** [review]
##### Automated comment from [CodeApprove ➜](https://codeapprove.com/pr/tiny-pilot/tinypilot/1585)


⏳ Approval Pending (1 unresolved comments)
_Approval will be granted automatically when all comments are resolved_

Cool, much simpler now!

LGTM

<hr />

In: **.circleci/continue_config.yml**:

```
> Line 16
    default: << pipeline.git.branch >>
```
We need to revert this change before merging.


<hr />

👀 @db39 it's your turn please take a look

**Review by db39 — COMMENTED** [review]
##### Automated comment from [CodeApprove ➜](https://codeapprove.com/pr/tiny-pilot/tinypilot/1585)



<hr />

In: **.circleci/continue_config.yml**:
Resolved
